modules/kobalt-plugin-api/src/main/kotlin/com/beust/kobalt/internal/IncrementalManager.kt

```diff
@@ -56,18 +56,18 @@ class IncrementalManager @Inject constructor(val args: Args, @Assisted val fileN
         val bi = BuildInfo(map.values.toList())
         val json = GsonBuilder().setPrettyPrinting().create().toJson(bi)
 
-        synchronized(BUILD_INFO_FILE) {
-            Files.write(Paths.get(fileName), json.toByteArray(Charset.defaultCharset()))
-        }
+        Files.write(Paths.get(fileName), json.toByteArray(Charset.defaultCharset()))
```

Comment:
Why remove this synchronization? It's the most important, this is where all the `save` calls converge.